.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -0,0 +1,71 @@
+name: Bug report
+description: Create a bug report to help us improve. Use this template if you encountered an issue while integrating with or implementing the APIs of this SDK.
+labels: [bug, triage-required]
+
+body:
+- type: checkboxes
+  attributes:
+    label:  Prerequisites
+    description: Please check the following items before logging a new bug report.
+    options:
+    - label: This is not a Security Disclosure, otherwise please follow the guidelines in [Security Policy](https://github.com/adobe/aepsdk-react-native/security/policy).
+      required: true
+    - label: I have searched in this repository's issues to see if it has already been reported.
+      required: true
+    - label: I have updated to the latest released version of the SDK and the issue still persists.
+      required: true
+
+- type: textarea
+  attributes:
+    label: Bug summary
+    description: Please provide a summary of the bug you are reporting.
+  validations:
+    required: true
+
+- type: textarea
+  attributes:
+    label: Environment
+    description: |
+      Please provide the Platform, OS version, SDK version(s) used, IDE version, and any other specific settings that could help us narrow down the problem.

```

Comment:
We can ask customers to run `npx react-native info` and provide output. 
Actually, we can also reuse some steps from the react-native repo's bug report template:
https://github.com/facebook/react-native/blob/main/.github/ISSUE_TEMPLATE/bug_report.yml